### PR TITLE
Added a nice page transition no docs page

### PR DESCRIPTION
**Pull request by babakfp**
## Description
https://user-images.githubusercontent.com/44144724/148620784-703e5eef-f95d-44c3-b115-0c447a4a4d69.mp4
I'm not sure about the code but it works😄. I didn't much think about the format, structure, and this kind of stuff so...

## Motivation and Context
The docs navigation was missing something🤷‍♂️

### CODE REVIEW

File: src/routes/docs/__layout.svelte [hunk 1]
```diff
@@ -193,26 +204,28 @@
...
-						rel="noreferrer noopener"
-						target="_blank"
-					>
-						Edit this page
-					</HyperlinkButton>
-				</div>
-			</header>
-			<slot />
-		</div>
+		<PageTransition refresh={pagePath}>
```
Comment (TheOnlyTails) [inline src/routes/docs/__layout.svelte:207]: Why is a wrapper element necessary here? Is there no way to put the transition on the `markdown-body` div?

File: src/lib/PageTransition.svelte [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<script>
+  import { fly } from "svelte/transition";
+  export let refresh = "";
+</script>
+
+{#key refresh}
+  <div
+		in:fly={{ y: 6, duration: 400, delay: 400 }}
```
Comment (TheOnlyTails) [inline src/lib/PageTransition.svelte:8]: This doesn't respect the `prefers-reduced-motion` setting.

**Review by TheOnlyTails — COMMENTED** [review]
I like how it looks, but A11y is lacking and the use of a wrapper element here is weird to me.

File: src/lib/PageTransition.svelte (hunk 2 above)
Comment (babakfp) [reply src/lib/PageTransition.svelte]: I never work with `prefers-reduced-motion`, so🤷‍♂️ There isn't a built-in solution, and adding extra libraries and stuff is just making me say whatever, I didn't even want animations anymore😄

**Comment by TheOnlyTails** [comment]
This is kind of unrelated, but if you could get the sidebar to auto-dismiss when navigating on mobile, that'd be great. That way the animation would still be visible.

**Comment by babakfp** [comment]
Yep, looks way much better

**Review by TheOnlyTails — APPROVED** [review]
LGTM! love this change!

**Comment by TheOnlyTails** [comment]
@Tropix126 @yaichenbaum if you like this, I'm good with merging right away.

**Comment by yaichenbaum** [comment]
> @Tropix126 @yaichenbaum if you like this, I'm good with merging right away.

Looks good

**Pull request merged**